### ISSUE TITLE
Build failing

### ISSUE THREAD

**Issue by juliotux** [astropy/astroscrappy#75]
In my CI tests that require building astroscrappy from sources, the build started failing today. The reported error is:

```
      ./astroscrappy/astroscrappy.c: In function ‘__pyx_import_star_set’:
      ./astroscrappy/astroscrappy.c:25148:20: error: lvalue required as left operand of assignment
      25148 |     CYTHON_ATOMICS = __Pyx_PyObject_IsTrue(o); if (unlikely((CYTHON_ATOMICS == (int)-1) && PyErr_Occurred())) __PYX_ERR(2, 26, __pyx_L2_error)
            |                    ^
      error: command '/usr/bin/gcc' failed with exit code 1
```

This problem only occurs when using Python 3.10. More details [here](https://github.com/sparc4-dev/astropop/runs/7550693455?check_suite_focus=true#step:8:581).

**Comment by DBerke** [comment]
Also experiencing what looks like the same crash on Python 3.7.13, and also started failing today.

**Comment by juliotux** [comment]
I just realized that Cython had a release today for [v0.29.31](https://github.com/cython/cython/releases/tag/0.29.31), almost at the same time my tests started fail. May be `astroscrappy` is not compatible with this newer version.

**Comment by juliotux** [comment]
It looks like several packages are broken, including `scipy`. There are more information in https://github.com/cython/cython/issues/4927 , but as described by [this comment](https://github.com/cython/cython/issues/4927#issuecomment-1197921021), the use of memoryviews and `from module import *` is breaking the building. Pin to `cython<=0.29.30` could fix the building until the bug is fixed.

**Comment by cmccully** [comment]
It looks like they are already almost ready to release a patch version. Should we wait for that? Or maybe if it isn't released in a day or pin the version?

**Comment by juliotux** [comment]
@cmccully yes, they created a [0.29.32 milestone](https://github.com/cython/cython/milestone/92) with these related bugs. So, we can imagine that they will release this fix soon.

**Comment by juliotux** [comment]
Fixed in 0.29.32 version, released today.
